pt/index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="pt">
+<head>
+  <title>Não pergunte para pedir, apenas peça</title>
+  <meta property="og:title" content="Não pergunte para pedir, apenas peça" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
```

Comment:
should be   <meta property="og:url" content="https://dontasktoask.com/pt" />

see for reference